app/views/tag/_profileCard.html.erb

```diff
@@ -1,41 +1,46 @@
 
-<div class="card-view">
+<div class="card-view profile-card">
```

Comment:
Oh wow so this separate template existed but you've made it match the standard topic card! Awesome!